lib/usecases/get_log_in_status_use_case.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter_survey/database/secure_storage.dart';
+import 'package:flutter_survey/usecases/base/base_use_case.dart';
+import 'package:injectable/injectable.dart';
+
+@Injectable()
+class GetLogInStatusUseCase extends SimpleUseCase<Future<bool>> {
```

Comment:
@Shayokh144 @Tuubz Do you think `GetLoginStatus` is a good name for `bool` data type return? What if `true` or `false`?

Reply:
Personally, I'd go for `CheckIsLoggedInUseCase` 🙏

Reply:
Works for me 👍